practices/open-source-support.md

```diff
@@ -3,20 +3,20 @@
 
 ## Versioning
 
-Our open source projects are versioned with [semver](http://semver.org/). You should read through the semver documentation if you're not familiar with it.
+Our open source projects are versioned with [semver](semver.md). You should read through the [semver documentation](http://semver.org) if you're going to release new versions.
 
 
-## Release process
+## Release process
 
-To publish a new version of one of our projects:
+To publish a new version of one of our open source projects:
 
-- Switch to the `master` branch, version commits are the only commits that shouldn't be in a [pull-request](code-review.md).
-- Increment either the major, minor, or patch version in the `package.json` file. If you're unsure which, have a chat about it or re-read the semver docs. Also verify if there are any mentions to the old version in the `README.md` or any other files, and update them too.
-- Add an entry to the `HISTORY.md` file outlining the changes in the new version. Take your time, this log should be useful to developers who use this project – it should help them make decisions about whether they can or should upgrade.
-- Commit your changes with a message formatted as `Version 1.2.3` – this helps people find version commits in the log.
-- Tag your newly created commit with the version number. E.g. `git tag 1.2.3`. Do *not* prepend a `v`, for example `v1.2.3`.
-- Push both the commit and the new tags to origin: `git push && git push --tags`. It's really important to push tags as well!
-- Depending of how the project is set up, the build system will automatically publish your new version based on the tag. Otherwise you may have to run `npm publish` for this to happen.
+* Switch to the `master` branch. Version commits are the only ones that shouldn't be committed to a feature branch.
+* Increment either the major, minor, or patch version in the `package.json` file. If you're unsure which, check with the rest of the team or re-read the semver docs. Also verify if there are any mentions to the old version in the `README.md` or any other files, and update them too.
+* Add an entry to the `HISTORY.md` file outlining the changes in the new version. Take your time, this log should be useful to developers who work on the project - it should help them make decisions about whether they can or should upgrade. If there are additional commits on the `master` branch since the last version, be sure to include a description of what they do in your history update - talk to the developers who committed them, if necessary.
```

Comment:
I actually think "developers who use this project" is more accurate than "developers who work on the project" since people often use tools (and benefit from that history for making upgrading decisions) but don't contribute to it directly.